packages/0x.js/package.json

```diff
@@ -114,6 +114,9 @@
         "uuid": "^3.1.0",
         "web3": "^0.20.0"
     },
+    "optionalDependencies": {
```

Comment:
Because `@0xproject/migrations` is a private package and it doesn't make sense to publish it.

Reply:
Does publishing the migrations package help us with developer on-boarding, such as removing the need to download a snapshot, they can deploy the exchange contracts etc themselves?

Does it allow us to get an entire InProcess TestRPC in the CodeSandbox?

Reply:
It could... Any team could import it and then call it with their own deployer setup (own RPC endpoint, networkId, etc...). Think we could consider doing that at some point I guess. Hadn't thought about external developers using it.